Components/Button.ux

```diff
@@ -0,0 +1,71 @@
+<Panel ux:Class="Material.Button" Alignment="Center" Enabled="true" Raised="true" Height="48" MinWidth="88" Padding="8,0" HitTestMode="LocalBoundsAndChildren" BackgroundColor="White" InkColor="#ccc">
+
+	<string ux:Property="Text" />
+	<bool ux:Property="Enabled" />
+	<bool ux:Property="Raised" />
+	<float4 ux:Property="BackgroundColor"/>
+	<float4 ux:Property="InkColor"/>
+
+	<Rectangle ux:Name="RaisedButton_body" ZOffset="1" Height="36" Padding="8,0" CornerRadius="2">
+		<SolidColor Color="{ReadProperty this.BackgroundColor}" />
+		<WhileTrue Value="{ReadProperty this.Raised}">
+			<Shadow ux:Name="raisedShadow" Distance="2" Size="3" />
+		</WhileTrue>
+	</Rectangle>
+
+	<Rectangle ux:Name="innerPanel" ZOffset="3" Height="{ReadProperty RaisedButton_body.Height}" Padding="{ReadProperty RaisedButton_body.Padding}" CornerRadius="{ReadProperty RaisedButton_body.CornerRadius}" ClipToBounds="true">
+		<Text Value="{ReadProperty this.Text}" Font="RobotoMedium" ux:Name="text" Alignment="Center" FontSize="14" />
+	</Rectangle>
+
+	<WhileTrue Value="{ReadProperty this.Enabled}" Invert="true">
+
+		<Change text.Opacity="0.26"/>
+		<Change text.TextColor="#000000"/>
+
+	</WhileTrue>
+
+	<WhileTrue Value="{ReadProperty this.Enabled}">
```

Comment:
You don't need to prefix these properties with an explicit `this.*`, since they are referring to the whole `ux:Class` we are currently in and `this.*` is implied.